docs/reference/master-configuration-reference.md

```diff
@@ -338,3 +338,104 @@ Comma-separated list of `pattern=N` settings for file-filtered logging.
 Default: *empty*
 
 Run-time configurable: yes
+
+## restrictions
+
+The following options specify the restrictions that can be applied by the
```

Comment:
One last comment. Should we call this **EXPERIMENTAL** in this release(?) There have been so many subtle details during the review process that we've certainly missed something :blush: WDYT?

Reply:
Yes, please. I will create another issue to cover those unseen details. If you have some bullet points in mind you can leave them here.